pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactory.java

```diff
@@ -567,6 +567,10 @@ private void parseSingleRuleNode(RuleSetReferenceId ruleSetReferenceId, RuleSetB
             rule.setUsesTypeResolution();
         }
 
+        if (hasAttributeSetTrue(ruleElement, "metrics")) {
```

Comment:
Good point! Is [this repo](https://github.com/pmd/pmd.sourceforge.net) the sources for `http://pmd.sourceforge.net/` ? In that case I think adding a line to [this xsd definition](https://github.com/pmd/pmd.sourceforge.net/blob/master/ruleset_2_0_0.xsd), which is loaded by rulesets,  would do the trick